src/helpers/FormValidation/FormValidation.js

```diff
@@ -349,7 +349,7 @@ class FormValidation {
     touchedField = {},
   } = {}) {
     return validateFieldsPerFieldset(
-      schema,
+      schema || {}, // If schema is null for some reason (schema request haven't fulfilled yet) we need to normalize it
```

Comment:
@sneridagh so if the schema hasn't loaded yet, we effectively just skip validation. Is that okay? I think so, since the validation that counts should happen on the server anyway.

Reply:
yeah, maybe that is better.